plot_types/stats/barbs.py

```diff
@@ -14,14 +14,13 @@
 np.random.seed(1)
 X = [[2, 4, 6]]
 Y = [[1.5, 3, 2]]
-U = -np.ones((1, 3)) * 0
-V = -np.ones((1, 3)) * np.linspace(50, 100, 3)
+U = np.zeros_like(X)
```

Comment:
This was made lowercase everywhere else. Why uppercase here?

Reply:
I mostly follow the capitalization in the signature of the respective plot function, some of which are lowercase, some are uppercase.

There's a loose convention that 1D arrays have regular lowercase names and 2D arrays from `np.meshgrid()` have uppercase names (e.g. `X, Y = np.meshgrid(x, y)`). I suspect that the signature roughly follows this pattern: If we expect 2D input, we use an uppercase parameter name. For `barbs` this is a bit ambiguous as parameters can be 1D or 2D, so I followed the signature style.